internal/host/installcmd.go

```diff
@@ -74,6 +74,7 @@ func (i *installCmd) GetSteps(ctx context.Context, host *models.Host) ([]*models
 		return nil, err
 	}
 
+	i.log.Info("Install command releaseImage: %s, mcoImage: %s", releaseImage, mcoImage)
```

Comment:
I would recommend to log the entire command. It will help debugging too

Reply:
What do you mean by "entire command" ?

Reply:
/hold

Reply:
The command line in the following line, including the podman invocation